### PR TITLE
Initial data updates

### PR DESCRIPTION
**Pull request by AriHrannar**
## Status
**Ready**

## Description
1. Updating initial data with new parliament members
2. Case and vote gather looks at data from last 3 parliament sessions
3. Fixing issues with creating parliament members automatically, seems like copy pasting from althingi.is caused issues with encoding.
4. Added a connection between parliament and parliament members

### CODE REVIEW

File: src/case_gather/cron_jobs.py [hunk 1]
```diff
@@ -15,8 +15,9 @@ class GatherCases(CronJobBase):
     def do(self):
         try:
             SentryLogger.warning('Starting case gather')
-            parliament_session = ParliamentSession.objects.latest('created')
-            CaseGatheringService.update_cases_by_session_number(parliament_session)
+            parliament_sessions = ParliamentSession.objects.all().order_by('-id')[:3]
```
Comment (Ragnar-H) [inline src/case_gather/cron_jobs.py]: does it make sense to pull `3` into a variable and share between the Cron jobs?

Something like `PARLIAMENTS_TO_LOOK_AT = 3` ?

File: src/case_gather/services.py [hunk 2]
```diff
@@ -61,9 +61,14 @@ def createCaseCreators(case_creator_names, case):
...
     """
     for case_creator_name in case_creator_names:
-        parliament_member = ParliamentMember.objects.get_or_create(
-            name=case_creator_name
-        )[0]
+        try:
+            parliament_member = ParliamentMember.objects.get(
+                name=case_creator_name
+            )
+        except ParliamentMember.DoesNotExist:
```
Comment (Ragnar-H) [inline src/case_gather/services.py]: could you do a log here?
I'm curious to see how often we'll run into this.
Would be nice to be able to dig into the logs later on

File: src/case_gather/cron_jobs.py (hunk 1 above)
Comment (AriHrannar) [reply src/case_gather/cron_jobs.py]: Yes that is a good idea

File: docker-compose.yml [hunk 3]
```diff
@@ -21,9 +21,9 @@ services:
       - DB_HOST=db
       - ALLOWED_HOSTS=["*"]
       - CORS_ORIGIN_WHITELIST=('localhost:3000',)
-      - GATHER_CASES_CRON_TIME_SECONDS=1440
-      - GATHER_VOTES_CRON_TIME_SECONDS=1440
-      - LINK_CASES_AND_PROMISES_CRON_TIME_SECONDS=1440
+      - GATHER_CASES_CRON_TIME_SECONDS=1
```
Comment (Ragnar-H) [inline docker-compose.yml]: did you revert this in a future commit?
Maybe keep these changes out of the commit history in the future?

File: src/case_gather/services.py (hunk 2 above)
Comment (AriHrannar) [reply src/case_gather/services.py]: There are a bunch of extra parliament members (varamenn) that sometimes come in and vote on issues, we would not have them in our system as we only have those directly elected

File: docker-compose.yml (hunk 3 above)
Comment (AriHrannar) [reply docker-compose.yml]: ya reverted

File: src/case_gather/cron_jobs.py (hunk 1 above)
Comment (AriHrannar) [reply src/case_gather/cron_jobs.py]: Ill implement and test this next tuesday

File: src/case_gather/services.py (hunk 2 above)
Comment (AriHrannar) [reply src/case_gather/services.py]: Might be a good idea to log here, as we need to add some data manually when this happens (party, district, abbreviation) as not everything is available to the scraper

File: src/parliament/services.py [hunk 4]
```diff
@@ -1,4 +1,11 @@
+import os
 from .models import ParliamentSession
 
 def parliament_session_ids_by_parliament(parliament):
     return ParliamentSession.objects.filter(parliament=parliament).values_list('id', flat=True)
+
+def get_parliament_sessions_to_look_at():
```
Comment (Ragnar-H) [inline src/parliament/services.py]: I like this :)

**Pull request merged**